account_mass_reconcile/tests/test_scenario_reconcile.py

```diff
@@ -28,12 +28,19 @@ def setUp(self):
         self.account_fx_expense_id = self.ref("account.income_fx_expense")
         self.acs_model = self.env['account.config.settings']
 
+        self.currency_eur_id = self.ref("base.EUR")
+        company = self.ref('base.main_company')
```

Comment:
You should call it `company_id` for clarity

Reply:
Done